wp-content/themes/haruoshi_origin/home.php

```diff
@@ -8,8 +8,18 @@
 get_header(); ?>
 
 <article class="container">
-	<section>
-		hoge
+	<section class="news content">
+		<h1 class="content__title">News</h1>
+		<?php
+			$posts_array = get_posts( array( 'post_type' => 'news' ) );
+			if (!empty($posts_array)):
+		?>
+		<?php foreach ( $posts_array as $post ) : setup_postdata( $post ); ?>
+			<div class="news__article">
+				<p><?php the_time( 'Y.m.d' )?></p>
+				<a class="news__article__title" href="<?php the_permalink() ?>"><?php the_title(); ?></a><br/>
```

Comment:
BEMではelement入れ子は禁止されているのでこんな感じになるかと思います。
```suggestion
				<a class="news__article-title" href="<?php the_permalink() ?>"><?php the_title(); ?></a><br/>
```

Reply:
done https://github.com/oooshima/origin_wordpress/pull/25/commits/a16de9b5e96d3e01a327756250ce81335e9ef668